lua/nvim-devdocs/init.lua

```diff
@@ -74,7 +95,7 @@ M.setup = function(opts)
   cmd("DevdocsInstall", M.install_doc, { nargs = "*", complete = completion.get_non_installed })
   cmd("DevdocsUninstall", M.uninstall_doc, { nargs = "*", complete = completion.get_installed })
   cmd("DevdocsOpen", M.open_doc, { nargs = "?", complete = completion.get_installed })
-  cmd("DevdocsOpenFloat", M.open_doc_float, { nargs = "?", complete = completion.get_installed })
```

Comment:
Did you mean to remove this line?

Reply:
I also don't get it

Reply:
> Did you mean to remove this line?

Yeaaah that was unintentional, I'm sorry. I made a redundant DevdocsOpenCurrentFileFloat, and when I removed that line it seems I accidentally removed the DevdocsOpenFloat one too. My apologies